hedyorg/hedy
# 🖊️ Improved existing error messages by highlighting keywords in error messages

Added keyword 'incomplete_command' and 'missing_command' as keywords that are translated and highlighted across different languages. Additionally changed existing error messages to use multiple commands in their message to be better understood, while also translated across multiple languages.

**How to test**

Follow these steps to verify this PR works as intended:

* launch http://0.0.0.0:8080
* change language to svenska and enter the following code in level 1:
`print lalalala`
`print`
* receive a translated error message instead of 'print':
![image](https://github.com/hedyorg/hedy/assets/71043418/db9383e6-1abf-4d5b-b07d-7ff224eccc4c)

*change language back to english and try the following code in level 3:
`animals is dog, cat, kangaroo`
`like is flowers`
`remove like`
`print animals at random`
* receive the following error message where 'remove' and 'from' are highlighted 
![image](https://github.com/hedyorg/hedy/assets/71043418/d0e63a15-3911-4746-a699-99ca633cabd2)

*try the following code in level 11:
`for no_in range 1 to 5`
* receive the following error message where 'for' and 'in' are highlighted
![image](https://github.com/hedyorg/hedy/assets/71043418/a6519b07-b091-43bd-b10e-d31ddf76ec12)
